src/views/admin/UserDetailsView.vue

```diff
@@ -30,9 +36,15 @@
   }
 
   onMounted(async () => {
-    currentPerson.value = await personStore.getPersonById(currentPersonId)
+    isLoading.value = true
+    try {
+      currentPerson.value = await personStore.getPersonById(currentPersonId)
+      isLoading.value = false
+    } catch (error: unknown) {
+      isLoading.value = false
+      errorMessage.value = t('errors.USER_DATA_LOADING_ERROR')
```

Comment:
Die Informationen `isLoading` und `errorCode` stehen im PersonStore. Bitte verwende die Werte aus dem Store, statt sie  in der Komponente zu duplizieren.

Reply:
Done.